libs/sam-formly-sample/src/lib/feature/formly-sample/formly.module.ts

```diff
@@ -1,15 +1,15 @@
 import { NgModule } from '@angular/core';
 import { CommonModule } from '@angular/common';
 import { FormsModule, ReactiveFormsModule } from '@angular/forms';
-import { SdsFormlyModule } from '../../../../../packages/sam-formly/src/lib/formly/formly.module';
 import { FormlyInputComponent } from './formly-input.component';
 import { FormlyModule } from '@ngx-formly/core';
+import { SdsFiltersModule } from '../../../../../packages/sam-formly/src/lib/formly-filters/sds-filters.module';

```

Comment:
can't we target @gsa-sam/sam-formly/src/...  instead?

Reply:
We can do that. Updated